@J0sh0nat0r when updating I'm having this issue :
"AssertionError: All storage sets must be assigned to a cluster" on all snuba consumers.

Have you also seen this ?

Thanks

--- a/sentry/templates/configmap-snuba.yaml
+++ b/sentry/templates/configmap-snuba.yaml
@@ -37,7 +37,7 @@ data:
             "host": env("CLICKHOUSE_HOST", {{ include "sentry.clickhouse.host" . | quote }}),
             "port": int(env("CLICKHOUSE_PORT", {{ include "sentry.clickhouse.port" . }})),
             "http_port": int(env("CLICKHOUSE_HTTP_PORT", 8123)),
-            "storage_sets": {"events", "migrations", "outcomes", "querylog", "sessions", "transactions"},
+            "storage_sets": {"events", "events_ro", "migrations", "outcomes", "querylog", "sessions", "transactions"},
             # FIXME: Snuba will be able to migrate multi node clusters in the future
             "single_node": env("CLICKHOUSE_SINGLE_NODE", "false").lower() == "true"
         },

